assets/stylesheets/_sections.scss

```diff
@@ -110,61 +140,160 @@
 }
 .leaderboard__item {
   font-family: $font-stack--retro;
-  font-size: $type--large;
+  font-size: $type--medium;
   color: $white;
+  @media screen and (min-width: $bp-small+100){
+    font-size: $type--large;
+  }
+}
+
+
+.blink{
+  color:$lemon;
+  -webkit-animation: blink linear 1.25s infinite;
+  animation: blink linear 1.25s infinite;
+}
+@-webkit-keyframes blink {
+  0% { opacity: 1; } 
+  50% { opacity: .1; } 
+  100% { opacity: 1; }
+}
+@keyframes blink {
+  0% { opacity: 1; } 
+  50% { opacity: .1; } 
+  100% { opacity: 1; }
 }
 
 .arcade-section__teaser {
   font-family: $font-stack--retro;
   font-size: $type--small;
   color: $white;
+  text-align: center;
 }
 
-.events-section {
-  align-items: center;
-  display: flex;
+.events-section{
+  background-color: #020710;
+  overflow: hidden;
 }
 
+
 .events-section__costume,
 .events-section__food {
   position: relative;
 }
-.events-section__image,
-.donations-section__image {
-  max-width: 360px;
+
+.events-section__food {
+  background: 
+    black
+    url("#{$asset-path}" + "#{$food-background}");
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  background-size: auto 100%;
+  width: 320px;
+  height: 320px;
+  @media screen and (min-width: $bp-small+50){
+    width: 400px;
+    height: 400px;
+  }
+  @media screen and (min-width: $bp-large+20){
+    width: 500px;
+    height: 500px;
+  }
+
+}
+.events-section__costume {
+  background: 
+    black
+    url("#{$asset-path}" + "#{$costume-background}");
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  background-size: auto 100%;
+  width: 320px;
+  height: 320px;
+  @media screen and (min-width: $bp-small+50){
```

Comment:
the inline media queries are interesting. 🤔 I know @miketlawrence advocates for this…I'm still on the fence, but it works here.

Reply:
i find them easier to pinpoint when im trying to affect the responsive style of a certain class